README.yaml

```
@@ -97,6 +117,42 @@ usage: |
               echo "${{ matrix.stack_slug }}"
   ```
 
+  ### Migrating from `v0` to `v1`
+  
+  `v1` replace `jq-query` input with `selected-filter` variable and simplify the query.
+  Now you need to specify only part used in select function of `jq-query`.  
+  
+  `v1` moved variables from `inputs` to atmos gitops config path `./.github/config/atmos-gitops.yaml`
+  
+  |         name             |
+  |--------------------------|
+  | `atmos-version`          |
+  | `atmos-config-path`      |
+  
+  
+  If you want the same behavior in `v1` as in `v0` you should create config `./.github/config/atmos-gitops.yaml` with the same variables as in `v1` inputs.
+  
+  ```yaml
+    - name: Selected Components
+      id: components
+      uses: cloudposse/github-action-atmos-terraform-select-components@v1
+      with:
+        atmos-gitops-config-path: ./.github/config/atmos-gitops.yaml
+        select-filter: '.settings.github.actions_enabled // false'
+  ```
+  
+  same behaviour as
+  
+  ```yaml
+    - name: Selected Components
+      id: components
+      uses: cloudposse/github-action-atmos-terraform-select-components@v0
+      with:
+        atmos-config-path: "${{ github.workspace }}/rootfs/usr/local/etc/atmos/"
+        jq-query: 'to_entries[] | .key as $parent | .value.components.terraform | to_entries[] | select(.value.settings.github.actions_enabled // false) | [$parent, .key] | join(",")'
+  ```
+
```

```suggestion
  For example, if you want the same behavior in `v1` as in `v0`, you should create the `./.github/config/atmos-gitops.yaml` config with the same variables as in `v1` inputs.
  
  In `v1`, the GItHub Action would invocation would look like this: 
  
  ```yaml
    - name: Selected Components
      id: components
      uses: cloudposse/github-action-atmos-terraform-select-components@v1
      with:
        atmos-gitops-config-path: ./.github/config/atmos-gitops.yaml
        select-filter: '.settings.github.actions_enabled // false'
  ```
  
  This is the same as the following `v0` configuration:
  
  ```yaml
    - name: Selected Components
      id: components
      uses: cloudposse/github-action-atmos-terraform-select-components@v0
      with:
        atmos-config-path: "${{ github.workspace }}/rootfs/usr/local/etc/atmos/"
        jq-query: 'to_entries[] | .key as $parent | .value.components.terraform | to_entries[] | select(.value.settings.github.actions_enabled // false) | [$parent, .key] | join(",")'
  ```

  Please note that the `atmos-gitops-config-path` is not the same file as as the `atmos-config-path`. 


```